tests/unit/apiPlatformService.test.js

```diff
@@ -198,12 +218,18 @@ describe('apiPlatformService', function () {
 
           fileSystemRepositoryStub.getFileHash.calledTwice.should.be.true();
 
-          result.should.be.an.Array();
-          result.length.should.equal(2);
-          result[0].path.should.equal(apiFiles[0].path);
-          result[0].hash.should.equal(fileHash);
-          result[1].path.should.equal(apiFiles[1].path);
-          result[1].hash.should.equal(fileHash);
+          should.deepEqual(result, [
```

Comment:
mmm apiFiles have more fields than path and audit and those fields are omitted in the response